airflow/api_fastapi/core_api/routes/public/dags.py

```diff
@@ -70,7 +72,13 @@ async def get_dags(
         SortParam,
         Depends(
             SortParam(
-                ["dag_id", "dag_display_name", "next_dagrun", "last_run_state", "last_run_start_date"],
+                {
+                    "dag_id": DagModel.dag_id,
+                    "dag_display_name": DagModel.dag_display_name,
+                    "next_dagrun": DagModel.next_dagrun,
+                    "last_run_state": DagRun.state,
+                    "last_run_start_date": DagRun.start_date,
+                },
```

Comment:
Most of this is not necessay. By default if the attribute is not in the mapping it will take it from the model. No need to duplicate it.

Reply:
Agree, I will put `allowed_attrs` back and remove the duplicate cases for `attr_mapping` in the `SortParm` refactoring PR.